docs/apis-tools/camunda-api-rest/camunda-api-rest-guidelines.md

```diff
@@ -0,0 +1,307 @@
+---
+id: camunda-api-rest-guidelines
+title: "Guidelines"
+description: "Learn about the basic guidelines, structures, and conventions of the Camunda 8 REST API."
+---
+
+Camunda follows a mix of proposed standards and best practices for RESTful design and consistent implementation across all components. This ensures customers working across all component APIs have a consistent and expected experience without having to study our API reference material or perform “validation testing” to see how our APIs respond.
+
+## Naming conventions
+
+Naming should be simple, intuitive, and consistent across Camunda 8 APIs. This improves the experience when working across multiple APIs by reducing friction.
```

Comment:
```suggestion

```

I would remove these sentences or change the tense. "Naming is simple, intuitive, ..."